graphql/src/graphql_schema.ml

```diff
@@ -1057,19 +1066,20 @@ end
           lifted_value >>| (function
           | Ok (value, errors) ->
               Ok ((name, value), errors)
-          | Error (`Argument_error _) as err ->
-              err
+          | Error (`Argument_error _) as err -> err
+          | Error (`Validation_error _) as err -> err
```

Comment:
These two patterns can be coalesced. For consistency, I'd also suggest `err` on it's own line like the other cases.

Reply:
This seems to work for me:

```ocaml
          | Error (`Argument_error _)
          | Error (`Validation_error _) as err ->
              err
```

Reply:
Oh turns out I didn't know the syntax correctly
you don't have to add `as err` on both lines
I'll fix that now